sos/report/plugins/microk8s.py

```diff
@@ -34,6 +34,9 @@ def setup(self):
 
         microk8s_subcmds = [
             'addons repo list',
+            'helm list -a',
+            'helm repo list',
+            'helm plugin list',
```

Comment:
While I understand the intent, we don't duplicate collections in `sos` - instead we aim to ensure the proper plugin is enabled by default in any environment that could reasonably want that to happen. In this case, that means ensuring that something in the `microk8s` environment is used as an enablement trigger in the new `helm` plugin (if it's possible to have `microk8s` installed without `helm` being present using vendor-support or commonly-documented installation methods).